### PR TITLE
harfbuzz: bump dependencies

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **harfbuzz/all**

Bump dependencies

---

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
Are you based on top of master?

**Comment by madebr** [comment]
@ericLemanissier 
Oh sorry, I'll rebase

File: recipes/harfbuzz/all/conanfile.py [hunk 1]
```diff
@@ -62,41 +62,40 @@ def configure(self):
...
             self.requires("icu/69.1")
         if self.options.with_glib:
-            self.requires("glib/2.69.1")
+            self.requires("glib/2.70.0")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version],
                   destination=self._source_subfolder, strip_root=True)
 
+    @functools.lru_cache(1)
```
Comment (ericLemanissier) [inline recipes/harfbuzz/all/conanfile.py:75]: very nice !

**Review by Croydon — CHANGES_REQUESTED** [review]
I think that you forgot to delete `example.c` (you copied instead of renamed?)

**Comment by madebr** [comment]
> I think that you forgot to delete `example.c` (you copied instead of renamed?)

Good catch.

While trying to get a recipe building on MSVC, I made a lot of changes to a few recipes.
After I got finished, I just copied the changes for each recipe on top of current master but forgot to remove those files I renamed earlier.

**Review by ericLemanissier — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**